lib/bundler/fetcher/downloader.rb

```diff
@@ -14,7 +14,7 @@ def fetch(uri, options = {}, counter = 0)
         raise HTTPError, "Too many redirects" if counter >= redirect_limit
 
         response = request(uri, options)
-        Bundler.ui.debug("HTTP #{response.code} #{response.message}")
+        Bundler.ui.debug("HTTP #{response.code} #{response.message} #{uri}")
```

Comment:
@segiddins Works for me, and I can make that change.  The only problem I do see with it is the URIs plus query params can be quite lengthy, so putting the URI first might make it hard to quickly find 500 errors if the response code/message are at the end.  Thoughts?